HW_4/task2.py

```diff
@@ -0,0 +1,9 @@
+'''Напишите функцию для транспонирования матрицы'''
+def trans_matrix(lst:list[list]):
+    row = zip(*lst)
+    return [list(r) for r in row]
```

Comment:
Строка не сильно удлинится, если написать все в одну
```suggestion
    return [list(r) for r in zip(*lst)]
```
Вот еще как можно
```suggestion
    row = zip(*lst)
    return [*map(list, zip(*lst))]
```